script/deploy

```diff
@@ -1,10 +1,9 @@
 #!/bin/bash
 
-
-git clone https://github.com/apache/incubator-servicecomb-docs.git
-
-gitbook build incubator-servicecomb-docs/java-chassis-reference users/servicecomb-java-chassis-doc
- 
-bundle exec jekyll build --config _config.yml
-
-bundle exec jekyll server --port 4000 --host 
+script_dir=$(cd `dirname $0`; pwd)
+echo cru$script_dir
+base_dir=`dirname ${script_dir}`
+echo build $base_dir
+cd ${base_dir}
+sudo bundle exec jekyll build --config _config.yml
+sudo bundle exec jekyll server --port 4000 --host 
```

Comment:
when running the server the host url is changed,  it is not for the asf-site to use.